338 docs rfc the other half of the docs (#4199)
timescale/docs: _partials/_migrate_live_migrate_faq_all.md

- Do not perform any explicit DDL operation on the source database during the course of migration.

> - If you are migrating from $SELF_LONG or MST, disable the chunk retention policy on your source database 

Reviewer: ```suggestion
- If you are migrating from $SELF_LONG or $MST_SHORT, disable the chunk retention policy on your source database 
```

   "(ALTER.*OWNER.*|GRANT|REVOKE)"  > /tmp/grants.psql 
   ```
   
1. Run `grants.psql` on your target $SERVICE_LONG. 
   ```shell
   psql -d $TARGET -f /tmp/grants.psql
   ```

> ### Postgres to Tiger Cloud: “live-replay not keeping up with source load”

Reviewer: ```suggestion
### Postgres to $CLOUD_LONG: “live-replay not keeping up with source load”
```

> 1. Go to the $CONSOLE -> Insights tab and find the query which takes significant time

Reviewer: ```suggestion
1. Go to $CONSOLE -> `Monitoring` -> `Insights` tab and find the query which takes significant time
```

2. If the query is either UPDATE/DELETE, make sure the columns used on the WHERE clause have necessary indexes.
3. If the query is either UPDATE/DELETE on the tables which are converted as hypertables, make sure the REPLIDA IDENTITY(defaults to primary key) on the source is compatible with the target primary key. If not, create an UNIQUE index source database by including the hypertable partition column and make it as a REPLICA IDENTITY. Also, create the same UNIQUE index on target.

> ### ERROR: out of memory (or) Failed on request of size xxx in memory context "yyy" on a Tiger Cloud service

Reviewer: ```suggestion
### ERROR: out of memory (or) Failed on request of size xxx in memory context "yyy" on a $SERVICE_LONG
```
